app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,191 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+$light_gray: #F2F2F2;
+$white: #FFF;
+$beach_blue: #5bc0de;
+
+body {
+  padding-top: 20px;
+}
+
+#index_background_header {
+  text-align: center;
```

Comment:
Properties should be ordered color, font-family, font-size, font-weight, padding-top, text-align